.circleci/config.yml

```diff
@@ -113,22 +116,35 @@ jobs:
             sudo systemctl start zookeeper
             sudo systemctl start kafka
       - run: 
-          name: start kafka connect
+          name: Start kafka connect
           command: |
             sudo mkdir -p /usr/local/share/kafka/plugins/
             sudo cp /home/circleci/repo/target/splunk-kafka-connect*.jar /usr/local/share/kafka/plugins/
-            sed -i 's/plugin\.path\=connectors\//plugin\.path\=\/usr\/local\/share\/kafka\/plugins\//' /home/circleci/repo/config/connect-distributed.properties
-            sed -i 's/bootstrap\.servers.*/bootstrap\.servers\=localhost\:9092/' /home/circleci/repo/config/connect-distributed.properties
-            sed -i 's/config\.storage\.replication\.factor.*/config\.storage\.replication\.factor\=1/' /home/circleci/repo/config/connect-distributed.properties
-            sed -i 's/offset\.storage\.replication\.factor.*/offset\.storage\.replication\.factor\=1/' /home/circleci/repo/config/connect-distributed.properties
-            sed -i 's/status\.storage\.replication\.factor.*/status\.storage\.replication\.factor\=1/' /home/circleci/repo/config/connect-distributed.properties
-            sudo /usr/local/kafka/bin/connect-distributed.sh -daemon /home/circleci/repo/config/connect-distributed.properties
-      - run: 
+            sed -i 's/plugin\.path\=connectors\//plugin\.path\=\/usr\/local\/share\/kafka\/plugins\//' /home/circleci/repo/config/connect-distributed-quickstart.properties
+            sudo /usr/local/kafka/bin/connect-distributed.sh /home/circleci/repo/config/connect-distributed-quickstart.properties
+          background: true
```

Comment:
oh there is this option? nice find!

Reply:
yes, this will start kafka connect in background